pages/coinbase/homepage.py

```diff
@@ -0,0 +1,30 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
```

Comment:
remove any imports you're not using anymore. (all imports you're not using, if it's more than just this one)